### PR TITLE
Add database

### PR DESCRIPTION
**Pull request by lvthang1606**
- [x] Add migration files to create the database tables following the requirement
- [x] Add migration file to create two roles
- [x] Add migration file to create an admin
- [x] Run the application with docker started
- [x] Make sure the database tables were created with the correct structure and relationships
- [x] Check the roles were created

### CODE REVIEW

File: src/main/resources/db/migration/DEV/V1_0_0_202303021434__add_sample_data.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+INSERT INTO roles(name)
+VALUES ('ADMIN'), ('CONTRIBUTOR');
+
+INSERT INTO users(username, password, firstName, lastName, enabled, avatar, role_id)
+VALUES ('admin01', 'encrypted_password', 'Thang', 'Le', true, 'avatar01.png', (SELECT id FROM roles WHERE name = 'ADMIN')),
```
Comment (nguyenvanman) [inline src/main/resources/db/migration/DEV/V1_0_0_202303021434__add_sample_data.sql]: Is this password usable?

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021433__initial_schema.sql [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+CREATE TABLE users
+(
+    id         UUID           NOT NULL  PRIMARY KEY  DEFAULT  uuid_generate_v4(),
+    username   VARCHAR(50)    NOT NULL,
+    password   VARCHAR(50)    NOT NULL,
+    firstName  VARCHAR(100),
+    lastName   VARCHAR(100),
+    enabled    BOOLEAN        NOT NULL,
+    avatar     VARCHAR(255),
+    role_id    UUID           NOT NULL,
```
Comment (nguyenvanman) [inline src/main/resources/db/migration/COMMON/V1_0_0_202303021433__initial_schema.sql:20]: Please add 2 more fields: createdAt and updatedAt

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021433__initial_schema.sql [hunk 3]
```diff
@@ -1,7 +1,37 @@
...
 CREATE TABLE books
 (
-    id   UUID         NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
-    name VARCHAR(255) NOT NULL
+    id           UUID          NOT NULL  PRIMARY KEY  DEFAULT  uuid_generate_v4(),
+    title        VARCHAR(255)  NOT NULL,
+    author       VARCHAR(255)  NOT NULL,
+    description  VARCHAR(255),
+    created_at   TIMESTAMP     NOT NULL,
+    updated_at   TIMESTAMP     NOT NULL,
```
Comment (nguyenvanman) [inline src/main/resources/db/migration/COMMON/V1_0_0_202303021433__initial_schema.sql]: This field can be nullable

**Review by nguyenvanman — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/DEV/V1_0_0_202303021434__add_sample_data.sql (hunk 1 above)
Comment (voquanghoa) [reply src/main/resources/db/migration/DEV/V1_0_0_202303021434__add_sample_data.sql]: Let's correct it later.

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021433__initial_schema.sql [hunk 4]
```diff
@@ -1,7 +1,37 @@
...
+    image        VARCHAR(255),
+    user_id	     UUID          NOT NULL,
```
Comment (voquanghoa) [inline src/main/resources/db/migration/COMMON/V1_0_0_202303021433__initial_schema.sql:35]: Wrong format

**Review by voquanghoa — APPROVED** [review]

**Pull request merged**